x-pack/plugins/ml/public/alerting/preview_alert_condition.tsx

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiCode,
+  EuiDescriptionList,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import type { AlertingApiService } from '../application/services/ml_api_service/alerting';
+import { MlAnomalyDetectionAlertParams, PreviewResponse } from '../../common/types/alerts';
+import { composeValidators } from '../../common';
+import { requiredValidator, timeIntervalInputValidator } from '../../common/util/validators';
+import { invalidTimeIntervalMessage } from '../application/jobs/new_job/common/job_validator/util';
+
+export interface PreviewAlertConditionProps {
+  alertingApiService: AlertingApiService;
+  alertParams: MlAnomalyDetectionAlertParams;
+}
+
+const AlertInstancePreview: FC<PreviewResponse['results'][number]> = React.memo(
+  ({ jobIds, timestampIso8601, score, topInfluencers, topRecords }) => {
+    const listItems = [
+      {
+        title: i18n.translate('xpack.ml.previewAlert.jobsLabel', {
+          defaultMessage: 'Job IDs:',
+        }),
+        description: jobIds.join(', '),
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.timeLabel', {
+          defaultMessage: 'Time: ',
+        }),
+        description: timestampIso8601,
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.scoreLabel', {
+          defaultMessage: 'Anomaly score:',
+        }),
+        description: score,
+      },
+      ...(topInfluencers && topInfluencers.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topInfluencersLabel', {
+                defaultMessage: 'Top influencers:',
+              }),
+              description: (
+                <ul>
+                  {topInfluencers.map((i) => (
+                    <li key={i.unique_key}>
+                      <EuiCode transparentBackground>{i.influencer_field_name}</EuiCode> ={' '}
+                      {i.influencer_field_value} [{i.score}]
+                    </li>
+                  ))}
+                </ul>
+              ),
+            },
+          ]
+        : []),
+      ...(topRecords && topRecords.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topRecordsLabel', {
+                defaultMessage: 'Top records:',
```

Comment:
Should I expect top records to be listed here when creating an influencer type alert? In this test, there are none found in the alert:

![image](https://user-images.githubusercontent.com/7405507/107363936-bc507d00-6ad2-11eb-9150-574cac36e676.png)

Does it use the threshold set in the alert for the `record_score` threshold?

Reply:
Same as the current watcher behaviour, I apply the score threshold for records `gte: 3`. I reckon we can remove filtering in this case for top record hits. What do you think?

Reply:
I think we can remove those score threshold `gte: 3 ` checks for the top records and top influencers, and list the top 3 by score, irrespective of whether they are `gte: 3`.

Reply:
Removed this threshold in 22783535287fdf9341f5b8bc135607e0253b0af4